docs/account/forceRefreshAccount.md

```diff
@@ -0,0 +1,17 @@
+**Usage**: `account.forceRefreshAccount()`    
+**Description**: Force a refresh of all the addresses informations (utxo, balance, txs...) by invalidating previous.   
+**Important**: This is used for experience developer. If you have an issue or need using it, please do not hesitate to post an issue on GitHub.   
+
+Parameters: 
```

Comment:
Looks like the following info is for the encrypt method

Reply:
Good catch!